src/AbstractProxyFixer.php

```diff
@@ -23,47 +24,71 @@
 abstract class AbstractProxyFixer extends AbstractFixer
 {
     /**
-     * @var FixerInterface
+     * @var FixerInterface[]
      */
-    protected $proxyFixer;
+    protected $proxyFixers;
 
     public function __construct()
     {
-        parent::__construct();
+        foreach (Utils::sortFixers($this->createProxyFixers()) as $proxyFixer) {
+            $this->proxyFixers[$proxyFixer->getName()] = $proxyFixer;
```

Comment:
for what I see, we don't need assoc array here

Reply:
Proxied fixers are sorted by priority, using names rather than some not so pedictable integer indexes is more convenient for main fixer to access them. It's used [here](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3149/files#diff-194fee79f2296fcded24ff97b2bb231fR48) for example.

Reply:
then it shall be typehinted as `array<string, FixerInterface>`

Reply:
Done.